Gemfile

```diff
@@ -1,11 +1,11 @@
 source 'https://rubygems.org'
 
-gem 'rails', '~> 4.0'
+gem 'rails', '~> 5.0'
```

Comment:
The custom in other apps is to pin rails to the minor version, maybe that would be good here too?